libbeat/docs/monitoring/monitoring-internal-collection.asciidoc

```diff
@@ -70,6 +70,27 @@ Stack Monitoring UI. To get a cluster's `cluster_uuid`,
 call the `GET /` API against that cluster.
 <2> This setting identifies the hosts and port numbers of {es} nodes
 that are part of the monitoring cluster.
++
+If you want to use PKI authentication to send monitoring events to
+{es}, you must specify a different set of configuration options. For
+example:
++
+["source","yml",subs="attributes"]
```

Comment:
```suggestion
[source,yaml]
```

Reply:
No need to complicate this with subs=attributes since the example doesn't contain any attributes that need to be resolved by asciidoctor.